Sources/BraintreePayPal/BTPayPalApprovalURLParser.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// The type of PayPal authentication flow to occur
+enum PayPalRedirectType {
```

Comment:
q: Would it make sense for this enum to be in it's own file?

Reply:
I opted to leave in here since it's so small, but down to move if folks feel strongly!